src/link.zig

```diff
@@ -484,6 +484,7 @@ pub const File = struct {
         // well-commented.
 
         const id_symlink_basename = "llvm-ar.id";
+        const artifact_dir_handle = if (base.options.module) |mod| mod.zig_cache_artifact_directory.handle else directory.handle; // If it doesn't have an artifact dir, just put it wherever the output is?
```

Comment:
Is this a question for the reader?
Please don't put comments on the side, put them above the code they refer to.